src/topology/bounded_continuous_function.lean

```diff
@@ -465,6 +465,20 @@ instance : add_comm_group (α →ᵇ β) :=
 @[simp] lemma coe_sub : ⇑(f - g) = λ x, f x - g x := rfl
 lemma sub_apply : (f - g) x = f x - g x := rfl
 
+open_locale big_operators
+@[simp] lemma coe_sum {ι : Type*} (s : finset ι) (f : ι → (α →ᵇ β)) :
+  ⇑(∑ i in s, f i) = (∑ i in s, (f i : α → β)) :=
+begin
+  ext a,
+  apply finset.induction_on s,
+  { simp, },
+  { intros i s nm h, simp [h, finset.sum_insert nm], },
+end
```

Comment:
There's a shorter proof like the one in #6375

Reply:
```suggestion
(add_monoid_hom.mk coe_fn coe_zero coe_add).map_sum f s
```
although it probably makes sense to add `coe_fn_add_hom` as #6375 does,

Reply:
Thanks Eric, I've added `coe_fn_add_hom` and use it now.